Controllers/ReviewsController.cs

```diff
@@ -0,0 +1,75 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Movies.Api.Models;
+using Movies.Api.Repositories;
+
+namespace Movies.Api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ReviewsController : ControllerBase
+{
+    private readonly IReviewRepository _reviewRepository;
+    private readonly IMovieRepository _movieRepository;
+    private readonly IUserRepository _userRepository;
+
+    public ReviewsController(IReviewRepository reviewRepository, IMovieRepository movieRepository, IUserRepository userRepository)
+    {
+        _reviewRepository = reviewRepository;
+        _movieRepository = movieRepository;
+        _userRepository = userRepository;
+    }
+
+    [HttpGet]
+    public IActionResult GetReviews()
+    {
+        var reviews = _reviewRepository.GetReviewsAsync().Result;
+
+        return Ok(reviews);
+    }
+
+    [HttpGet("by_movie/{id}")]
```

Comment:
For better API design concepts the path should be `/movies/{id}/reviews`

Reply:
True. I moved it to the movies controller